python/singa/tensor.py

```diff
@@ -938,7 +938,69 @@ def mult(A, B, C=None, alpha=1.0, beta=0.0):
         singa.MultWithScale(alpha, A.singa_tensor, B.singa_tensor,
                             beta, C.singa_tensor)
         return C
-
+def tensormult (A,B,axes=2):
+    try:
```

Comment:
pls add some comments for this function. it is difficult to understand the logic.